proposals/inline-arrays.md

```diff
@@ -91,15 +91,15 @@ element_access
 
 The *argument_list* of an *element_access* is not allowed to contain `ref` or `out` arguments.
 
-An *element_access* is dynamically bound ([§11.3.3](expressions.md#1133-dynamic-binding)) if at least one of the following holds:
+An *element_access* is dynamically bound ([§11.3.3](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/expressions.md#1133-dynamic-binding)) if at least one of the following holds:
```

Comment:
Fixed in #7368

> By the way, is there a feature request for Microsoft Learn to rewrite this kind of link

Not right now. I may make updates to the tool so that we use custom anchors without the clause numbers. That would be more resilient. I don't know if that will work, because of different clauses with the same name. "General" is an obvious example. I don't know if concatenating all parent clauses would make them unique. I'll run some tests.